etc/cas.properties

```diff
@@ -38,6 +38,7 @@ osf.url=http://localhost:5000/
 osf.institutionLogin.url=http://localhost:5000/login/?campaign=institution
 osf.resendConfirmation.url=http://localhost:5000/resend/
 osf.forgotPassword.url=http://localhost:5000/forgotpassword/
+osf.createAccount.url=http://localhost:5000/login/
```

Comment:
Just a reminder for deployment, we'll need to add this in the stage/prod configs.